zerver/lib/test_helpers.py

```diff
@@ -1,8 +1,9 @@
 from __future__ import absolute_import
-from typing import Any, Callable, Generator, Iterable, Tuple
+from typing import Any, Callable, Generator, Iterable, Tuple, Sized, Union, Optional # NOQA
```

Comment:
You don't need to add `NOQA` tags -- we cull unnecessary imports only periodically rather than checking for it in CI, and we'd need to add those to every `typing` import since typing is never actually used at runtime, so we can just exclude the typing stuff from our culling as a general rule :)